### DISCUSSION TITLE
oci-based helm charts in kustomize

### DISCUSSION THREAD

**Discussion by snuggie12** [renovatebot/renovate#27815] [Suggest an Idea]
### Tell us more.

We get an `UNSUPPORTED_PROTOCOL` error when trying to pull a helm chart with an `oci://` repo in `kustomization.yaml`

Having looked at other issues/discussions, it looks to me that this can be achieved by updating https://github.com/renovatebot/renovate/blob/main/lib/modules/manager/kustomize/extract.ts. If all that is required is something similar to argocd where `repo` is manipulated and using the docker datasource (plus some test-related stuff,) I could give it a try (I have never worked with typescript.)

Am I correct to presume a custom manager and a manager can't have the same `fileMatch`? Our testing wasn't exhaustive, but we tried to apply a custom manager to hold us over before the kustomize manager was updated.

**Comment by rarkins** [comment]
PRs welcome

Custom managers can have the same fileMatch

**Reply by viceice** [reply]
Now we've two open PR's 🙃 
- #27163
- #27899
